main.js

```diff
@@ -18,72 +17,68 @@ function toggleMenu() {
+}
 
-  const username = document.getElementById("username");
-  const email = document.getElementById("email");
-  const errorMessage = document.getElementById("error");
-  const message = document.getElementById("msg");
+function handleSubmit(e) {
+  const username = document.getElementById('username');
+  const email = document.getElementById('email');
+  const errorMessage = document.getElementById('error');
+  const message = document.getElementById('msg');
 
-  if (username.value === "") {
-    errorMessage.textContent = "Name cannot be blank";
-    errorMessage.classList.add("show");
-    e.preventDefault()
-    return;
-    
-  }else{
-    errorMessage.classList.remove("show");
+  if (username.value === '') {
+    errorMessage.textContent = 'Name cannot be blank';
+    errorMessage.classList.add('show');
+    e.preventDefault();
+    return false;
   }
 
   if (username.value.length < 3 || username.value.length > 20) {
-    errorMessage.textContent = "Name must have between 3 to 20 characters";
-    errorMessage.classList.add("show");
-    e.preventDefault()
-    return;
-
-  }else{
-    errorMessage.classList.remove("show");
+    errorMessage.textContent = 'Name must have between 3 to 20 characters';
+    errorMessage.classList.add('show');
+    e.preventDefault();
+    return false;
   }
 
   if (checkUppercase(email.value)) {
-    errorMessage.textContent = "Most enter a valid email in lower case";
-    errorMessage.classList.add("show");
-    e.preventDefault()
-    return;
+    errorMessage.textContent = 'Most enter a valid email in lower case';
+    errorMessage.classList.add('show');
+    e.preventDefault();
+    return false;
+  }
 
-  }else{
-     errorMessage.classList.remove("show");
+  if (message.value.length > 1000 || message.value === '') {
+    errorMessage.textContent = 'Most write a message of less than 1000 characters';
+    errorMessage.classList.add('show');
+    e.preventDefault();
+    return false;
   }
+  errorMessage.classList.remove('show');
 
+  const obj = {
+    fullname: username.value,
+    emailAddress: email.value,
+    message: message.value,
+  };
 
-  if (message.value.length > 1000 || message.value === "") {
-    errorMessage.textContent =
-      "Most write a message of less than 1000 characters";
-    errorMessage.classList.add("show");
-    e.preventDefault()
-    return;
+  localStorage.setItem('DATA', JSON.stringify(obj));
+  return true;
+}
+
+form.addEventListener('submit', handleSubmit);
 
-  }else{
-    errorMessage.classList.remove("show");
+const getData = localStorage.getItem('DATA');
+const getDataValue = JSON.parse(getData);
+
+window.addEventListener('load', () => {
+  if (localStorage.getItem('DATA')) {
+    document.querySelector('#username').value = getDataValue.fullname;
+    document.querySelector('#email').value = getDataValue.emailAddress;
+    document.querySelector('#msg').value = getDataValue.message;
   }
-  // document.getElementById("form").submit()
-}
+});
```

Comment:
- [ ] Actually, the changes you made are not related to the one requested by the previous reviewer, you have to make sure that input values are stored locally even if the form is not submitted. This means that you'll target each input value and save it to the localStorage the way that even if the user left the page and come back, the form will be filled with the previous data. Kindly, implement these changes. 

Hint: 
1. Store inputs values in an object. eg: 
  ```
     const formData = {
       name: nameInput.value,
       email: emailInput.value,
       body: bodyInput.value,
     };
  ```
2. Use `localStorage.setItem` and `localStorage,getItem` to set and get data.
3. On window load get the store data from the local storage.